app/assets/javascripts/articles.js

```diff
@@ -1,13 +1,21 @@
 App.articleQueue.onIncoming = function(data) {
-  var alertDiv = $('#floating-alert');
+  var alertDiv = document.querySelectorAll('#floating-alert');
 
   if (alertDiv.length === 0) {
```

Comment:
But if you change `querySelectorAll()` to `getElementById()`, you'd probably need a different conditional here.